pilot/docker/Dockerfile.proxyv2

```diff
@@ -13,11 +13,14 @@ COPY gcp_envoy_bootstrap.json /var/lib/istio/envoy/gcp_envoy_bootstrap_tmpl.json
 
 RUN chown -R istio-proxy /var/lib/istio
 
+RUN apt-get update && apt-get install --no-install-recommends -y libcap2-bin && apt-get clean && rm -rf  /var/log/*log /var/lib/apt/lists/* /var/log/apt/* /var/lib/dpkg/*-old /var/cache/debconf/*-old
```

Comment:
if we are going to do this we should move this into the base image, not run apt-get here. Lets get consensus on this before moving down that path though

Reply:
Yup, fine for me